catch exceptions and region chooser added

c0decave opened the pull request:
tools runs now fine with current libs and you can even choose which region to check for ressources.

Commit:
catchting faults when non authorized, quick and dirty

Commit:
quick add

Commit:
added region chooser

Commit:
show regions

rmogull reviewed (commented):
Changes look great... but my laptop died and it will take me another couple days to get back from travel to test and approve. Thanks!

Commit:
region,show-regions,save-json,fin

c0decave commented:
Glad you like it. I added some other small changes, like show-regions, changed flag from --cregion to --region and a flag for saving output as json. Oh and added output for regions as well. Thats it :)
Happy c0ding!
